tool/webqa-process-schemaorg.js

```diff
@@ -962,7 +963,11 @@ class SchemaProcessor {
                 name: `${this._className ? this._className + ' in ' : ''}Schema.org`,
                 description: 'Scraped data from websites that support schema.org'
             },
-            impl: {}
+            impl: {
+                whitelist: new Ast.Value.Array(
+                    this._white_list.map((q) => new Ast.Value.String(q.trim().toLowerCase()))
```

Comment:
Why is it lowercased, and not case-sensitive? If it's developer-specified, you can match the case of the query.

Reply:
this is only because we have been using lowercase previously... i can remove that from all other scripts too.

Reply:
addressed